signer/src/config/mod.rs

```diff
@@ -234,6 +235,11 @@ pub struct SignerConfig {
     /// The number of signatures required for the signers' bootstrapped
     /// multi-sig wallet on Stacks.
     pub bootstrap_signatures_required: u16,
+    /// The number of seconds the coordinator will wait
+    /// before processing a new Bitcoin block
+    /// (allowing it to propagate to the others signers)
+    #[serde(deserialize_with = "duration_seconds_deserializer")]
+    pub bitcoin_processing_delay: std::time::Duration,
```

Comment:
I think i would make the config explicitly in seconds, with a validation ceiling of 300 or so (half the expected time of a bitcoin block); that'll keep things simpler.

Reply:
Nevermind, I read too quickly, you're doing this already 😅

Reply:
Do you mean renaming the variable to `... _seconds`? Can add the ceiling, no problem.

Reply:
No I actually meant what you are doing, I missed your custom deserializer at a glance and briefly thought that this was expected to be provided as a textual duration 🙈

Yeah I think ensuring that we have a reasonable max that's within our expected bitcoin block time would be wise

Reply:
So I implanted it because I read this too late (4e85213). I am not 100% sure we want to have that cap though.

Reply:
Yeah I'm not sure on the number, but I can't think of a situation where we would ever want to exceed 15-30 secs, so this is probably okay as a failsafe just to make sure that the signer can never wait _longer_ than a typical bitcoin block (i.e. waiting into the next block).